packages/fxa-settings/src/components/PageSecondaryEmailAdd/index.tsx

```diff
@@ -17,6 +17,8 @@ export const PageSecondaryEmailAdd = (_: RouteComponentProps) => {
   const [email, setEmail] = useState<string>();
   const inputRef = useRef<HTMLInputElement>(null);
   const { l10n } = useLocalization();
+
+  const subtitleText = l10n.getString('add-secondary-email-step-1');
```

Comment:
We probably want some fallback text in case `l10n.getString` fails?

Reply:
FWIW, I don't think we have been very consistent with setting fallback strings.